frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -72,12 +77,21 @@ export function Editor({
             NotebookNodePerson,
             NotebookNodeFlag,
             SlashCommandsExtension,
+            NotebookNodeTimestamp,
+            Extension.create({
+                addGlobalAttributes() {
+                    return [
+                        {
+                            types: [NotebookNodeType.Recording],
+                            attributes: { notebookShortId: { default: notebookId } },
```

Comment:
Hmmmm not sure that I love this... Could we not just use the notebookLogic instead? Every node is mounted within a notebookLogic so you should be able to easily grab the currently mounted notebookLogic and pull out the Id from that?

If I've missed something we can look at it together